src/app/objective.ts

```diff
@@ -97,6 +97,7 @@ export class ImmutableObjective {
     this.groups = o.groups;
     this.tags = o.tags;
     this.assignments = o.assignments;
+    this.assignments.length > 0;
```

Comment:
This line doesn't do anything. Was it left here by mistake?

Reply:
Sorry, fixed it now.